src/components/transportation-icon/TransportationIcon.tsx

```diff
@@ -38,7 +42,8 @@ export const TransportationIcon: React.FC<TransportationIconProps> = ({
     size == 'small'
       ? styles.transportationIcon_small
       : styles.transportationIcon;
-
+  const minWidth = wide ? styles.transportationIcon_wide : undefined;
+  const rightMargin = withRightMargin ? styles.rightMargin : undefined;
```

Comment:
Components shouldn't handle their own margin, they should be added from the outside by using a `style` parameter.